### DISCUSSION TITLE
Why force a DOM structure that doesn't need to exist in v-select?

### DISCUSSION THREAD

**Discussion by larrykkk** [vuetifyjs/vuetify#14883] [General]
I'm using vuetify's v-select but in fact I don't need to show the hint at all, why don't you choose to hide the DOM (.v-text-field__details and margin-bottom at .v-input__slot) above the hint position by default?

![image](https://user-images.githubusercontent.com/25131952/160272088-fbbe9e74-5008-408c-ba57-47d370b555e4.png)

**Comment by KaelWD** [comment]
Consistency.
